api/v1beta1/condition_consts.go

```diff
@@ -152,6 +152,9 @@ const (
 	// MachineHasFailureReason is the reason used when a machine has either a FailureReason or a FailureMessage set on its status.
 	MachineHasFailureReason = "MachineHasFailure"
 
+	// RemediateMachineReason is the reason used when a machine has either a ManualRemediationReason or a ManualRemediationMessage set on its status.
+	RemediateMachineReason = "MachineManualRemediation"
```

Comment:
Naming to be finalized

Reply:
The value should be consistent with the name of the constant

Reply:
Similar to what I wrote on the annotation. I would keep this more generic. So I would use e.g. `HasRemediateMachineAnnotation`. We should not assume that someone set this annotation manually.
(same for the labels_and_annotations.md and some other places in this PR)

The description is very confusing to me, maybe outdated?

Reply:
Adjusted to

`HasRemediateMachineAnnotationReason = "HasRemediateMachineAnnotation"`